### PR TITLE
Flash Toast

### PR DESCRIPTION
**Pull request by khailq-1062**
https://edu-redmine.sun-asterisk.vn/issues/32592

### CODE REVIEW

**Comment by khailq-1062** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -15,3 +15,14 @@ def display_error object, method, name
   error = "#{name} #{object.errors.messages[method][0]}"
   content_tag :div, error, class: "error-feedback"
 end
+
+def toastr_flash type
+  case type
+  when "danger"
```
Comment (longcn-1045) [inline app/helpers/application_helper.rb]:
```suggestion
  when :danger
```

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -15,3 +15,14 @@ def display_error object, method, name
...
+    "toastr.error"
+  when "success"
```
Comment (longcn-1045) [inline app/helpers/application_helper.rb]: .

File: app/views/shared/_toastr.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<div id="toastr">
+  <% if flash.present? %>
+    <%= content_tag :script do %>
+      <% flash.each do |type, message| %>
+        <%= toastr_flash "#{type}" %>('<%= message %>')
```
Comment (anhnt-1044) [inline app/views/shared/_toastr.html.erb]: "

**Comment by khailq-1062** [comment]
ready

**Pull request merged**